lib/MiLight/MiLightRadioInterface.h

```diff
@@ -0,0 +1,29 @@
+
+#ifdef ARDUINO
+#include "Arduino.h"
+#else
+#include <stdint.h>
+#include <stdlib.h>
+#endif
+
+#ifndef MILIGHTRADIOINTERFACE_H_
+#define MILIGHTRADIOINTERFACE_H_
+
+class MiLightRadioInterface
```

Comment:
Good idea, - I went after prove of concept - feel free mock around with the code.